lib/api-create.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const { API } = require('@janiscommerce/api');
+const Settings = require('@janiscommerce/settings');
+const MongoDBIndexCreator = require('@janiscommerce/mongodb-index-creator');
+
+const mongoDBIndexCreator = new MongoDBIndexCreator();
+
+const ClientModel = require('./model-client');
+
+class ClientCreateAPI extends API {
+
+	async validate(data = this.data) {
+
+		if(data === null || typeof data !== 'object' || Array.isArray(data))
+			throw new Error('Invalid data: Should be an object, also not an array.');
+
+		if(!Array.isArray(data.clients))
+			throw new Error('Invalid data: Should have a clients property and must be an array.');
+	}
+
+	async process({ clients } = this.data) {
+
+		const clientModel = new ClientModel();
+
+		const databaseSettings = Settings.get('database')[clientModel.databaseKey];
+
+		const clientDatabase = {
+			dbHost: databaseSettings.host
+		};
+
+		if(databaseSettings.protocol)
+			clientDatabase.dbProtocol = databaseSettings.protocol;
+
+		if(databaseSettings.port)
+			clientDatabase.dbPort = databaseSettings.port;
+
+		if(databaseSettings.user)
+			clientDatabase.dbUser = databaseSettings.user;
+
+		if(databaseSettings.password)
+			clientDatabase.dbPassword = databaseSettings.password;
+
+
+		const clientsCodes = [];
+		const clientsToCreate = clients.map(client => {
+			clientsCodes.push(client.code);
+			return {
+				code: client,
+				status: ClientModel.statuses.active,
+				...clientDatabase,
+				dbDatabase: `janis-${client}`
+			};
+		});
+
+		await Promise.all([
+			clientModel.multiSave(clientsToCreate),
+			mongoDBIndexCreator.executeForClientDatabases()
+		]);
+		await this.postSaveHook(clientsCodes);
+	}
+
+	postSaveHook() {
+		return true;
+	}
```

Comment:
```suggestion
	postSaveHook(clientCodes) {
		return clientCodes;
	}
```

Adicionalmente agregaria **JSDocs** para este método para indicar que el método exista y como se usar (y salga en los editores y ayudé a quienes lo vayan a usar)